src/index.js

```diff
@@ -1,11 +1,23 @@
 var bodyParser = require('body-parser');
 var app = require('express')();
 var handleWfhRequest = require('./handle-wfh-request');
+var parseText = require('./parse-text')
 
 app.use(bodyParser.urlencoded({ extended: true }));
 app.listen(process.env.PORT);
 
 app.post('/wfh', (req, res) => {
 	res.send();
-	handleWfhRequest(req.body.user_id, req.body.response_url);
+	if (parseText.checkIfClear(req.body.text))
```

Comment:
Okay, my understanding was a bit flawed then, thanks for clarifying. I'm okay with what you have then. We'll just need to update the command usage documentation in Slack once this merges.